### DISCUSSION TITLE
Feature Request: Add "before" and "after" text of the values

### DISCUSSION THREAD

**Discussion by abid112** [andreruffert/range-slider-element#91] [Q&A] [answered]
Currently there is no way to add custom text before/after the value. That would be great if you can add this.

**Comment by andreruffert** [comment, marked as answer]
@abid112 from my point of view your described "Feature Request" is already possible. 
Displaying the value does not get handled by the custom element and is completely up to you.

This [codepen](https://codepen.io/andreruffert/pen/JjjjBbd) might help.
